package.json

```diff
@@ -2,6 +2,7 @@
   "name": "cn.ubuntu.com",
   "description": "Django website for cn.ubuntu.com",
   "scripts": {
+    "start": "yarn run build && concurrently --raw 'yarn run watch-css' 'yarn run watch-js' 'yarn run serve'",
```

Comment:
Pretty sure `watch-css` and `watch-js` don't currently exist... they'll need creating for this to pass.

Reply:
Well the issue I was seeing is that this has already been done in another PR https://github.com/canonical-web-and-design/cn.ubuntu.com/pull/409. Can we revert it and then we can add it to @steverydz 's PR? This is what we have done in the others